src/components/RoomCard.tsx

```diff
@@ -48,7 +47,7 @@ const ResponsiveArea = (
 export const RoomCard: React.FC<{
   room: RoomTypes;
   roomId: string;
-  offer: OfferRecord;
+  offer: Offer;
```

Comment:
`OfferRecord` should be kept. Offers that arrives from API request do not have Ids inside the object. OfferRecord is the Offer with id. (see img)    
![Screenshot 2022-08-16 at 18 17 39](https://user-images.githubusercontent.com/47505482/184928967-42a1fa48-bebf-4b19-98fb-2742a76e4b4b.jpg)